pyproject.toml

```diff
@@ -1,84 +1,26 @@
 [tool.poetry]
-name = "langgraph"
-version = "0.0.69"
-description = "langgraph"
+name = "langgraph-monorepo"
```

Comment:
```
Warning: The current project could not be installed: No file/folder found for package langgraph-monorepo
...

If you want to use Poetry only for dependency management but not for packaging, you can disable package mode by setting package-mode = false in your pyproject.toml file.
In a future version of Poetry this warning will become an error!
```

Reply:
nice, fixed!

Reply:
ugh, unfortunately it doesn't work w/ older versions of poetry, will make a TODO to bump poetry version in our CI and then fix up